snapcraft/models/project.py

```diff
@@ -449,7 +449,7 @@ def _validate_bus_name(cls, name):
     )
     @classmethod
     def _validate_time(cls, timeval):
-        if not re.match(r"^[0-9]+(ns|us|ms|s|m)*$", timeval):
+        if not re.match(r"^[0-9]+(ns|us|s|m)*$", timeval):
```

Comment:
Given that these are timeout strings that get passed unaffected to snapd and [parsed](https://github.com/canonical/snapd/blob/79632c61a54e38f988bb452e8022c691cb203e5c/timeout/timeout.go#L60) using [golang's `time.ParseDuration`](https://pkg.go.dev/time#ParseDuration), I believe a more correct version of this regex would be:

```suggestion
        if not re.match(r"^([0-9]+(ns|us|ms|s|m)){1,5}$" timeval):
```